lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -135,19 +152,25 @@ try {
         if ($frequency -eq "once") {
             $trigger = New-ScheduledTaskTrigger -Once -At $time
         }
+        elseif ( $frequency -eq "interval"){
+            $trigger = New-ScheduledTaskTrigger -Once -RepetitionInterval (iex $("new-timespan -$interval_unit $interval")) -At $time -RepetitionDuration ([timespan]::MaxValue)
```

Comment:
I've been messing around with this script a bit and I think I found a potential problem here. At least in my testing, having `-RepetitionDuration ([timespan]::MaxValue)` as a flag in windows server 2016 (probably windows 10 too) breaks the script with a formatting error when the command generates the xml. Not having the flag for windows server 2012 breaks it as well because that version requires a flag. My solution was:
```
If (([System.Environment]::OSVersion.Version).Major -eq 10){
    $trigger = New-ScheduledTaskTrigger -Once -RepetitionInterval (iex $("new-timespan -$interval_unit $interval")) -At $time
} Else {
    $trigger = New-ScheduledTaskTrigger -Once -RepetitionInterval (iex $("new-timespan -$interval_unit $interval")) -At $time -RepetitionDuration ([timespan]::MaxValue)
}
```
I'm sure there's a better way of doing that, but I'm not too familiar with powershell so that's what I found in a quick google search. Excuse my ignorance if I missed something obvious here (maybe my powershell version in 2012 was too old or something).